translations/README.id.md

```diff
@@ -83,4 +83,4 @@ cd React-GitHub-Resume
 
 ## 📄 License
```

Comment:
is that same in Indonesia?? for varification

Reply:
fixed it, thank you again for the correction